src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java

```diff
@@ -270,7 +283,7 @@ private String nodeLookup(String id, Node node) {
     }
 
     private String nodeLookup(String id, String label, String prop, Object value) {
-        return "(" + id + ":" + quote(label) + "{" + quote(prop) + ":" + toString(value) + "})";
+            return id + ":" + quote(label) + "{" + quote(prop) + ":" + toString(value) + "})";
```

Comment:
shouldn't it be consistent, i.e. if you remove the starting parenthesis, then also move the closing one to the other place?